src/views/Dashboard/Customer/Home/MyTest/index.vue

```diff
@@ -21,28 +23,140 @@
+                      height="25px"
+                      width="50%"
+                      dark
+                      color="primary"
+                      :to="{ name: 'order-history-detail', params: item}"
+                    ) Detail
+                    
+                    Button(
+                      v-if="item.status == 'Registered'"
+                      height="25px"
+                      width="50%"
+                      dark
+                      color="secondary"
+                      @click="goToInstruction(item.service_info.dna_collection_process)"
+                    ) Instruction
+
+                    Button(
+                      v-if="item.status == 'ResultReady'"
+                      height="25px"
+                      width="50%"
+                      dark
+                      color="secondary"
+                      @click="goToStakeData"
+                    ) Add as Bounty
+
+                template(v-slot:[`item.status`]="{item}")
+                  .customer-my-test__status
+                  span(:style="{color: setStatusColor(item.status)}") {{ item.status }}
           v-tab-item
             StakingServiceTab
 
+      modalBounty(
+        v-if="isBounty"
+        title="Do you want to add your test result as a data bounty?"
+        sub-title="You can learn more about data bounty by seeing the information"
+        link="/"
+        :loading="isLoading"
+      )
+        .modal-bounty__cta.d-flex.mt-8.justify-center
+          Button(outlined color="secondary" width="100" @click="cancelBounty") Cancel
+          Button(color="secondary" width="100" @click="submitBounty") Yes
 </template>
 
 <script>
-import { layersIcon, noteIllustration } from "@/common/icons"
+import { layersIcon, noteIllustration, medicalResearchIllustration } from "@/common/icons"
 import StakingServiceTab from "./StakingServiceTab.vue"
+import DataTable from "@/common/components/DataTable"
+import Button from "@/common/components/Button"
+import { mapState } from "vuex"
+import {
+  // searchOrder,
+  ordersByCustomer,
+  getOrdersData
+} from "@/common/lib/polkadot-provider/query/orders"
+import { queryLabsById } from "@/common/lib/polkadot-provider/query/labs"
+import { queryServicesById } from "@/common/lib/polkadot-provider/query/services"
+import localStorage from "@/common/lib/local-storage"
+import dataTesting from "./dataTesting.json"
+import modalBounty from "../DataBounty/modalBounty"
+import {
+  COVID_19,
+  DRIED_BLOOD,
+  URINE_COLLECTION,
+  FECAL_COLLECTION,
+  SALIVA_COLLECTION,
+  BUCCAL_COLLEVTION
+} from "@/common/constants/instruction-step.js"
 
 export default {
   name: "MyTest",
 
-  components: { StakingServiceTab },
+  components: {
+    StakingServiceTab,
+    DataTable,
+    Button,
+    modalBounty
+  },
 
   data: () => ({ 
     layersIcon, 
     noteIllustration, 
     cardBlock: false,
     documents: null,
-    tabs: null
+    tabs: null,
+    showing: false,
+    isReady: true,
```

Comment:
do we need this? please remove some unnecessary data

Reply:
done, deleted unnecessary data